widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterView.kt

```diff
@@ -80,6 +87,21 @@ class MessageCenterView(
         // Is needed to overlap existing app bar in existing view with this view's app bar.
         ViewCompat.setElevation(this, 100.0f)
         readTypedArray(attrs, defStyleAttr, defStyleRes)
+
+        handleInstChanges()
+    }
+
+    private fun handleInstChanges() {
```

Comment:
NIT: letter "e" is missing, isn't it? `handleInstChanges()` => `handleInsetChanges()`